app/models/thread.php

```diff
@@ -75,4 +75,63 @@ public static function countAll()
         $db = DB::conn();
         return (int) $db->value("SELECT COUNT(*) FROM thread");
     }
+
+    public function edit($thread_id)
+    {
+        if (!$this->title) {
+            throw new RecordNotFoundException('Record Not Found');
+        }
+
+        try {
+            $db = DB::conn();
+            $db->begin();
+            $db->query('UPDATE thread SET title = ?, created = NOW()
```

Comment:
Why set created to NOW? date created and date updated must be in separate columns. :bow: